syntax/jinja.vim

```diff
@@ -33,13 +33,7 @@ if g:jinja_syntax_html
   if version < 600
     so <sfile>:p:h/html.vim
   else
-    let ext = expand('%:e')
-    if ext !~ 'htm\|nunj|jinja\|j2' &&
-          \ findfile(ext . '.vim', $VIMRUNTIME . '/syntax') != ''
-      execute 'runtime! syntax/' . ext . '.vim'
-    else
-      runtime! syntax/html.vim
-    endif
+    runtime! syntax/html.vim
```

Comment:
Why did you take out all the other code here?

Reply:
I literally just replaced the entire file with a copy/paste of https://github.com/pallets/jinja/blob/master/ext/Vim/jinja.vim, so that's a question for upstream (I think.)

Reply:
ah, we already pulled in the latest change. But it looks like starting at commit 2d1439ca8242e6b328a6250943371d2f26ca5b16 we've diverged a little from the original source. So I won't accept this part of the pull request.

Reply:
:ok: